marklogic-data-hub/src/test/resources/flow-runner-test/flows/testFlow.flow.json

```diff
@@ -29,6 +29,10 @@
       "customHook": {},
       "options": {
         "permissions": "data-hub-operator,read,data-hub-operator,update"
+      },
+      "fileLocations": {
```

Comment:
Had to add this so that when the flow is run without a project, the input path can be made absolute.